gk/bpf.c

```diff
@@ -110,6 +110,7 @@ struct gk_bpf_pkt_frame {
 	struct flow_entry	*fe;
 	struct ipacket          *packet;
 	struct gk_config	*gk_conf;
+	bool			called_encapsulate;
```

Comment:
The naming here is OK but a little off, since in the case where we are forwarding a granted packet directly to the destination, it is valid but has not been encapsulated.

Reply:
I agree, but I don't have a better name. In the future, a packet may be encapsulated but forwarded to another destination that is not a grantor or the destination to enable load balancing. Any suggestion for this name?

Reply:
Maybe something like `ready_to_tx`? I guess the function `gk_bpf_encapsulate()` should really be called `gk_bpf_prep_for_tx()` or something.

Reply:
I like it. Thank you.